README.md

```diff
@@ -12,6 +12,7 @@ in a project.
 
 | Version | [hapi.js](https://github.com/hapijs/hapi) | Node  | Release Notes                                                 |
 | ------- | ----------------------------------------- | ----- | ------------------------------------------------------------- |
+| `v11.x` | `>=18.3.1 @hapi/hapi` && >=16.0 @hapi/joi | `>=8` |                                                               |
```

Comment:
I think the **@hapi/hapi** minimum version is `v18.4.0`

Reply:
👍